src/Console/Command/Init.php

```diff
@@ -14,7 +14,7 @@
     name: 'init',
     description: 'Initializes a new Castor project',
 )]
-class InitCommand extends Command
+class Init extends Command
```

Comment:
Why `Init` rather than `InitTask` ? This allows you to say that it is an initiation task. `Init` may seem too generic WDYT ?

Reply:
Yeah I thought about it but I was wondering how to rename the `TaskCommand`  then and didn't find a consistent naming for the 3 of them that wouldn't be confusing.

Maybe `RepackTask`, `InitTask` and `BaseTask`?

Reply:
Yes, I think it would be more coherent to name them that way 😄

Reply:
IMO these are internal code so it should keep respecting Symfony standards and be named command.

Reply:
@pyrech I thought about it too but I'm afraid this implicit understanding will still generate ambiguity and confusion, even for experienced contributors. Case in point: https://github.com/jolicode/castor/pull/218#discussion_r1446018497

Since castor is deeply coupled with symfony console and command, we could do `TaskSymfonyCommand`, `InitSymfonyCommand`, `RepackSymfonyCommand` ? Or `TaskConsoleCommand` etc. ? 

To really reflect which "command" it's about ?